api.planx.uk/webhooks/sanitiseApplicationData/index.ts

```diff
@@ -15,9 +15,12 @@ export const sanitiseApplicationData = async (
   next: NextFunction
 ): Promise<Response | void> => {
   const operations = getOperations();
-  let results: OperationResult[] = [];
+  const results: OperationResult[] = [];
   try {
-    results = await Promise.all(operations.map(operationHandler));
+    for (const operation of operations) {
+      const result = await operationHandler(operation)
+      results.push(result)
+    }
```

Comment:
Speed of operation doesn't really matter as it's a cron job, but we can no longer run these operations concurrently as we rely on `lowcal_session.data` for the `deleteApplicationFiles` operation to work.